clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -4086,6 +4086,7 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     // We want to compile sycl kernels.
     CmdArgs.push_back("-fsycl");
     CmdArgs.push_back("-fsycl-is-device");
+    CmdArgs.push_back("-fdeclare-spirv-builtins");
```

Comment:
Driver changes look good to me.  Any reason why set this in the driver as opposed to in the FE?

Reply:
Technically the change is not specific to SYCL so I designed it separately from SYCL and enable the feature via the driver.